UI/js-src/lsmb/menus/MenuRestStore.js

```diff
@@ -0,0 +1,119 @@
+define(["dojo/_base/declare",
+    "dojo/store/JsonRest", "dojo/store/Observable",
+    "dojo/store/Memory", "lsmb/menus/Cache",
+    "dijit/Tree", "dijit/tree/ObjectStoreModel",
+    "dijit/Menu", "dijit/MenuSeparator", "dijit/PopupMenuItem",
+    "dojo/when", "dojo/dom", "dojo/ready"
+], function(declare, JsonRest, Observable,
+    Memory, lsmbStoreCache,
+    Tree, ObjectStoreModel,
+    Menu, MenuSeparator, PopupMenuItem,
+    when, dom, ready
+){
+  return declare(
+    "lsmb/menus/MenuRestStore", [Menu], {
+    postCreate: function() {
+        // set up the store to get the tree data, plus define the method
+        // to query the children of a node
+        var restStore = new JsonRest({
+            target:      "menu.pl?action=menuitems_json",
+            idProperty: "id",
+            _getTarget: function(id){
+                return this.target;
+            },
+        });
+        var memoryStore = new Memory({idProperty: "id"});
+        var store = new lsmbStoreCache(restStore, memoryStore);
```

Comment:
Why can't we use the same model here as we used for the accounts and parts: simply initialize a MemoryStore from the JSON response from the server?
That eliminates the need to have a caching store.